src/interface/viewport.h

```diff
@@ -38,23 +38,24 @@ enum {
 	VIEWPORT_FLAG_HIDE_BASE = (1 << 12),
 	VIEWPORT_FLAG_HIDE_VERTICAL = (1 << 13),
 	VIEWPORT_FLAG_INVISIBLE_SPRITES = (1 << 14),
-	VIEWPORT_FLAG_15 = (1 << 15)
+	VIEWPORT_FLAG_15 = (1 << 15),
+	VIEWPORT_FLAG_SEETHROUGH_PATHS = (1 << 16)
 };
 
 enum {
-	VIEWPORT_INTERACTION_ITEM_NONE,
-	VIEWPORT_INTERACTION_ITEM_TERRAIN,
-	VIEWPORT_INTERACTION_ITEM_SPRITE,
-	VIEWPORT_INTERACTION_ITEM_RIDE,
-	VIEWPORT_INTERACTION_ITEM_WATER,
-	VIEWPORT_INTERACTION_ITEM_SCENERY,
-	VIEWPORT_INTERACTION_ITEM_FOOTPATH,
-	VIEWPORT_INTERACTION_ITEM_FOOTPATH_ITEM,
-	VIEWPORT_INTERACTION_ITEM_PARK,
-	VIEWPORT_INTERACTION_ITEM_WALL,
-	VIEWPORT_INTERACTION_ITEM_LARGE_SCENERY,
-	VIEWPORT_INTERACTION_ITEM_LABEL,
-	VIEWPORT_INTERACTION_ITEM_BANNER,
+	VIEWPORT_INTERACTION_ITEM_NONE = 0,
+	VIEWPORT_INTERACTION_ITEM_TERRAIN = 1,
+	VIEWPORT_INTERACTION_ITEM_SPRITE = 2,
+	VIEWPORT_INTERACTION_ITEM_RIDE = 3,
+	VIEWPORT_INTERACTION_ITEM_WATER = 4,
+	VIEWPORT_INTERACTION_ITEM_SCENERY = 5,
+	VIEWPORT_INTERACTION_ITEM_FOOTPATH = 6,
+	VIEWPORT_INTERACTION_ITEM_FOOTPATH_ITEM = 7,
+	VIEWPORT_INTERACTION_ITEM_PARK = 8,
+	VIEWPORT_INTERACTION_ITEM_WALL = 9,
+	VIEWPORT_INTERACTION_ITEM_LARGE_SCENERY = 10,
+	VIEWPORT_INTERACTION_ITEM_LABEL = 11,
+	VIEWPORT_INTERACTION_ITEM_BANNER = 12,
```

Comment:
It made sense in older versions of VS2015 to put in the numbers as it was annoying trying to double check the numbers. It now tells you when hovering over the value so it isn't really required. Thats one of the reasons I have sprinkled a few numbers in when implementing the track functions.